include/moq/client.h

```diff
@@ -118,7 +118,7 @@ namespace moq {
          *
          * @param metrics           Copy of the connection metrics for the sample period
          */
-        virtual void MetricsSampled(const ConnectionMetrics&& metrics);
+        virtual void MetricsSampled(const ConnectionMetrics metrics);
```

Comment:
Should this be const ref?

Reply:
It's a copy, but it could be a ref but the metrics are reset after calling this.

Reply:
changed.